crypto/src/conversation/mod.rs

```diff
@@ -43,6 +43,7 @@ use crate::{
 
 mod commit_delay;
 pub mod decrypt;
+mod durability;
```

Comment:
Maybe move the `#[cfg(test)]` here instead of `#![cfg(test)]` in `durability.rs`?